components/pricing_modal/content.tsx

```diff
@@ -151,7 +151,7 @@ function Content(props: ContentProps) {
     const hasLimits = hasSomeLimits(limits);
 
     const starterBriefing = [
-        formatMessage({id: 'pricing_modal.briefing.free.recentMessageBoards', defaultMessage: 'Access to {messages} most recent messages, {boards} most recent board cards'}, {messages: formatNumber(fallbackStarterLimits.messages.history), boards: fallbackStarterLimits.boards.cards}),
+        formatMessage({id: 'pricing_modal.briefing.free.recentMessageBoards', defaultMessage: 'Access to {messages} most recent messages, No board limits'}, {messages: formatNumber(fallbackStarterLimits.messages.history)}),
```

Comment:
This is the text in the description of the ticket, but I think the text there is to inform us internally of what the change is. I think the desired user-facing wording is in the google doc, and is to have `Unlimited board cards` as its own bullet point (https://docs.google.com/document/d/1i9iuyyV-PyuhIw19gxnCEbFfe6IE2fnWOoMThiz926o)